curriculum/challenges/english/03-front-end-libraries/redux/use-const-for-action-types.english.md

```diff
@@ -35,8 +35,8 @@ tests:
   - text: The <code>authReducer</code> function should handle multiple action types with a switch statement.
     testString: getUserInput => assert((function() { return typeof authReducer === 'function' && getUserInput('index').toString().includes('switch') && getUserInput('index').toString().includes('case') && getUserInput('index').toString().includes('default') })());
   - text: <code>LOGIN</code> and <code>LOGOUT</code> should be declared as <code>const</code> values and should be assigned strings of <code>LOGIN</code>and <code>LOGOUT</code>.
-    testString: getUserInput => assert((function() {  const noWhiteSpace = getUserInput('index').toString().replace(/\s/g,''); return (noWhiteSpace.includes('constLOGIN=\'LOGIN\'') || noWhiteSpace.includes('constLOGIN="LOGIN"')) && (noWhiteSpace.includes('constLOGOUT=\'LOGOUT\'') || noWhiteSpace.includes('constLOGOUT="LOGOUT"')) })());
-  - text: The action creators and the reducer should reference the <code>LOGIN</code> and <code>LOGOUT</code> constants.
+    testString: getUserInput => assert((function() { const noWhiteSpace = code.replace(/\s/g, ''); assert(/constLOGIN=('"`)LOGIN\1/.test(nonWhiteSpace) && /constLOGOUT=('"`)LOGOUT\1/.test(nonWhiteSpace)) }());
+    - text: The action creators and the reducer should reference the <code>LOGIN</code> and <code>LOGOUT</code> constants.
```

Comment:
You were not supposed to change the `text` line.  Please commit the following change to revert it back to its original state.
```suggestion
  - text: The action creators and the reducer should reference the <code>LOGIN</code> and <code>LOGOUT</code> constants.
```